WSLBlobNFS/wsl2_linux_script.sh

```diff
@@ -44,11 +124,12 @@ function install_samba ()
 function onetime_samba_setup ()
 {
     # Set username as the password
-    echo "Note: Samba password for $1 is same as the username. This makes it easier to mount shares."
+    vecho "Note: Samba password for $1 is same as the username. This makes it easier to mount shares."
     # remove "Added user $1" message
     echo -ne "$1\n$1" | tee - | smbpasswd -a -s $1
```

Comment:
why tee?
we can directly echo to stdin of smbpasswd.
Also, isn't "$1\n$1\n" safer, since smbpasswd expects enter to be pressed after username and password both.